sys/mips/mips/exception.S

```diff
@@ -95,7 +95,7 @@
 #ifdef CPU_CHERI
 #include <machine/cheriasm.h>
 #include <machine/cherireg.h>
-.set cheri_sysregs_accessible	# Don't warn on access to c27-c30
+.set cheri_sysregs_accessible	# Don't warn on access to c27-c30 ???
```

Comment:
The ??? is that I'm not sure if this is still relevant.

Reply:
I think it's been long enough to make the the default and remove the option.